openfoodfacts/api.py

```diff
@@ -450,6 +450,21 @@ def __init__(
         self.facet = FacetResource(self.api_config)
         self.robotoff = RobotoffResource(self.api_config)
 
+    def search_advanced(self, query: str, filters: dict = None, page: int = 1):
```

Comment:
Invalid type for `filters` parameter here:
```suggestion
    def search_advanced(self, query: str, filters: Optional[dict] = None, page: int = 1):
```